lms/djangoapps/instructor_task/tests/test_tasks.py

```diff
@@ -344,6 +344,8 @@ def test_overriding_success(self):
                 'lms.djangoapps.instructor_task.tasks_helper.module_state.get_module_for_descriptor_internal'
         ) as mock_get_module:
             mock_get_module.return_value = mock_instance
+            mock_instance.max_score = MagicMock(return_value=99999.0)
```

Comment:
Needed for test. As py2 mock automatically assigns a huge number if not mocked properly and type casted into int or float. which py3 assigns none. tests fail. So assigned a dummy huge number to pass test.